Resources/Prototypes/Roles/Jobs/Civ14/TDM/soviet.yml

```
@@ -0,0 +1,264 @@
+    shoes: civ13_shoes_Soviet_sapogi_boots
+    jumpsuit: civ13_uniform_Soviet_uniform
+    gloves: ClothingHandsGlovesFingerless
+    outerClothing: civ13_suit_Soviet_coat
+    head: civ13_head_soviet_ushanka_1
+# inhand:
+  #  - civ13_boltaction_Mosin
+
+# Machinegunner
+
+- type: job
+  id: SovietMachinegunner 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)